PD configuration file: add log level (#6302)
pingcap/docs: pd-configuration-file.md

Configuration items related to log

### `level`

> + The log level, which can be specified as "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"

Reviewer: ```suggestion
+ The log level. The value options are "DEBUG", "INFO", "WARNING", "ERROR", and "CRITICAL".
```

> + The log level. The value options are "DEBUG", "INFO", "WARNING", "ERROR", and "CRITICAL".

Reviewer: ```suggestion
+ The log level. The value options are "DEBUG", "INFO", "WARNING", "ERROR", and "CRITICAL".
```
